test/helpers/fixture_deploy_helper.rb

```diff
@@ -0,0 +1,90 @@
+module FixtureDeployHelper
+  # use deploy_fixture_set if you are not adding to or otherwise modifying the template set
+  def deploy_fixture_set(set, subset=nil, wait: true)
+    source_dir = fixture_path(set)
+    return deploy_dir(source_dir) unless subset
+
+    target_dir = Dir.mktmpdir
+    files = []
+    each_k8s_yaml(source_dir, subset) do |basename, ext, content|
+      Tempfile.open([basename, ext], target_dir) do |f|
+        files << f
+        f.write(content)
+      end
+    end
+
+    deploy_dir(target_dir, wait: wait)
+  ensure
+    files.each { |f| File.delete(f) } if files
+  end
+
+  # use load_fixture_data + deploy_loaded_fixture_set to have the chance to add to / modify the template set before deploy
```

Comment:
I'm happy to do this, but I haven't written RDoc comments before and I'm having trouble finding the standards for usage examples. Can you point me in the right direction tomorrow?